envergo/moulinette/views.py

```diff
@@ -130,6 +129,14 @@ def get_context_data(self, **kwargs):
             settings.VISITOR_COOKIE_NAME, ""
         )
 
+        if self.should_activate_optional_criteria():
+            if "moulinette" in context:
+                context["optional_forms"] = self.get_optional_forms(
+                    context["moulinette"]
+                )
+            else:
+                context["optional_forms"] = self.get_optional_forms()
```

Comment:
Moins verbeux, mais peut etre moins explicite.
```suggestion
            context["optional_forms"] = self.get_optional_forms(context.get("moulinette", None))
```

Reply:
Top, merci.